### PR TITLE
Add table booking page

### PR DESCRIPTION
**Pull request by gretamas**

### CODE REVIEW

File: pages/book-table.html [hunk 1]
```diff
@@ -3,7 +3,108 @@
...
+            <button class="primary-btn mr-60">
+              <a href="../pages/book-table.html" class="caps-text f-s32"
+                >Order now</a
+              >
+            </button>
+          </li>
+        </div>
+      </ul>
+    </nav>
+    <header></header>
```
Comment (ilona-liashenko) [inline pages/book-table.html:49]: looks like it's unnecessary tag here, maybe you just forgot to delete

Comment (ilona-liashenko) [inline pages/book-table.html]: looks like header is unnecessary tag here, I suppose you just forgot to delete it

File: pages/book-table.html [hunk 2]
```diff
@@ -3,7 +3,108 @@
...
+            class="book-table__input-lastname"
+            type="text"
+            name="last name"
+          />
+          <label class="ff-arial f-s36 fw400" for="email">Email</label>
+          <input class="book-table__input-email" type="email" name="email" />
+          <label class="ff-arial f-s36 fw400" for="phone">Phone</label>
+          <input class="book-table__input-phone" type="tel" name="phone" />
+        </form>
+        <div class="book-table__textarea">
```
Comment (ilona-liashenko) [inline pages/book-table.html]: I can be mistaken, but I would recommend to put textfield and button tags inside form tag as this ensures that when the user enters some text to all fields and submits the form by clicking the button, the whole data will be submitted together

**Review by ilona-liashenko — APPROVED** [review]

File: pages/book-table.html [hunk 3]
```diff
@@ -3,7 +3,108 @@
...
+          </li>
+        </div>
+      </ul>
+    </nav>
+    <header></header>
+    <main>
+      <section class="book-table">
+        <h1 class="book-table__title ff-arial f-s40 fw700">Book Table</h1>
+        <form class="book-table__form">
+          <label class="ff-arial f-s36 fw400" for="first name"
```
Comment (lizazp92) [inline pages/book-table.html:54]: I would say the font is super small comparing to the input size

File: pages/book-table.html [hunk 4]
```diff
@@ -3,7 +3,108 @@
...
+            >First Name</label
+          >
+          <input
```
Comment (elishab-dev) [inline pages/book-table.html]: a suggestion to add validation to all inputs. for example, 'required, min/max ammount of symbols to each input' etc

File: pages/book-table.html [hunk 5]
```diff
@@ -3,7 +3,108 @@
...
+            class="book-table__input-firstname"
+            type="text"
+            name="first name"
```
Comment (elishab-dev) [inline pages/book-table.html]: pls check the 'name' and 'for' attributes. it should not be separated with space if there are 2 words, but rather a hyphen. 
    "Use lowercase Latin letters.
    Separate different words by a hyphen.
    Do not use periods as well as numbers at the beginning of the name."

File: pages/book-table.html [hunk 6]
```diff
@@ -3,7 +3,108 @@
...
+          <label class="ff-arial f-s36 fw400" for="email">Email</label>
+          <input class="book-table__input-email" type="email" name="email" />
+          <label class="ff-arial f-s36 fw400" for="phone">Phone</label>
+          <input class="book-table__input-phone" type="tel" name="phone" />
+        </form>
+        <div class="book-table__textarea">
+          <label class="ff-arial f-s36 fw400" for="text box"
+            >Additional Notes</label
+          >
+          <textarea name="Notes" id="" cols="30" rows="10"></textarea>
```
Comment (elishab-dev) [inline pages/book-table.html]: the id is empty

**Review by elishab-dev — CHANGES_REQUESTED** [review]
these are suggestions!

File: pages/book-table.html (hunk 4 above)
Comment (elishab-dev) [reply pages/book-table.html]: I would also say the font (in all inputs) is super small comparing to the input size. what do you think?

File: pages/book-table.html (hunk 2 above)
Comment (gretamas) [reply pages/book-table.html]: Thank you Ilona! done

**Review by elishab-dev — APPROVED** [review]

**Pull request merged**